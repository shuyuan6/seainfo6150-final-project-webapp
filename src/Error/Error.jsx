import React from 'react'
import styles from "./Error.module.css"
import error from "../images/error.png"

const Error = () => {
    return (
        <div className = {styles.container}>
            <h1>The page you are looking for is not found!</h1>
            
            <img src={error} alt="Error" />
            <br />
            <p>You can click <a  href="/seainfo6150-final-project-webapp">here</a> to go to the home page!</p>

        </div>
    )
}

export default Error
