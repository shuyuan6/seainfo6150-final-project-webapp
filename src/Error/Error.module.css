.container {
  background: white;
  gap: 20px;
  font-family: Arial, Helvetica, sans-serif;
  padding: 25px 80px 80px 80px;
  border: 1px solid lightskyblue;
  justify-items: center;
  text-align: center;
}

img {
  max-width: 100%;
  height: auto;
}
