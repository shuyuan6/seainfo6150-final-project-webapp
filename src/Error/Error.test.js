import React from "react";
import { render } from "@testing-library/react";
import Error from "./Error.jsx";

it("renders the Error component correctly", () => {
  const { container } = render(<Error />);
  expect(container).toMatchSnapshot();
});
