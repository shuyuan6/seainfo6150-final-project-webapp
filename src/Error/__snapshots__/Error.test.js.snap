// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Error component correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      The page you are looking for is not found!
    </h1>
    <img
      alt="Error"
      src="error.png"
    />
    <br />
    <p>
      You can click 
      <a
        href="/"
      >
        here
      </a>
       to go to the home page!
    </p>
  </div>
</div>
`;
